dipper/sources/Orphanet.py

```diff
@@ -190,160 +169,3 @@ def _process_diseasegene(self, limit):
 
         return
 
-    def add_gene_to_disease(
-            self,
-            association_type,
-            gene_id,
-            gene_symbol,
-            disease_id,
-            eco_id):
-        """
-        Composes triples based on the DisorderGeneAssociationType element:
-        AND the suffixes:
-
-            - "gene phenotype"
-            - "function consequence"
-            - "cell origin"
-
-        xmlstarlet sel -t  -v "/JDBOR/DisorderList/Disorder/DisorderGeneAssociationList/
```

Comment:
I think we should keep this command somewhere in the script, perhaps in a different docstring

Reply:
Ah okay, just added it below the first docstring for now, so at least it's not lost to the ages...